CMakeLists.txt

```diff
@@ -1,10 +1,9 @@
-cmake_minimum_required(VERSION 2.8.3)
+cmake_minimum_required(VERSION 3.1.3)
```

Comment:
any special reason to increase the version of `cmake`?

Reply:
[CMAKE_CXX_STANDARD](https://cmake.org/cmake/help/v3.1/variable/CMAKE_CXX_STANDARD.html) was introduced by cmake 3.1 (which has only one release available: 3.1.3)